### PR TITLE
purge uppercase in prototypes names

### LINKED ISSUE #23708: Prototype names need to be made lower case.
we have a lot of uppercased ones that have made it through. there are some exceptions but mostly they should all be lower case

### PR DESCRIPTION
**Pull request by lzk228**
## About the PR

title
fixes #23708 

## Why / Balance

emisse said 🙏
image

## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

nuh

### CODE REVIEW

File: Resources/Prototypes/Actions/types.yml [hunk 1]
```diff
@@ -210,16 +210,16 @@
 
 - type: entity
   id: ActionChangeVoiceMask
-  name: Set name
-  description: Change the name others hear to something else.
+  name: set name
+  description: change the name others hear to something else.
```
Comment (lzk228) [inline Resources/Prototypes/Actions/types.yml:214]: man how did that squeeze here

**Comment by TheShuEd** [comment]
I think Uppercase name on action entities is ok

**Comment by Mervill** [comment]
I'm not a maintainer so I could be wrong but; this pr is too big to be reasonably reviewed and submitted. While it might be true that all entity names _should_ be lowercase, simply using (I assume) an automated process to correct this could lead to false positives and subtle bugs that may not be visible by simply reading the yml files.

I doubt you went though these prototypes individually and tested to make sure the name changes do not actually break anything.

We should probably do this over many smaller PRs (where each PR is probably a group of related prototypes) so that it's easier to actually test and confirm that nothing is breaking as a result.

This will likely create headaches for downstreams if thousands of prototypes suddenly have different names, even if the difference is just their capitalization.

Like I said though, I'm not a maintainer. Maybe we can just say lgtm and smash that merge button.

**Comment by IProduceWidgets** [comment]
Changing the names shouldn't cause issues anywhere I can think of? if a system is for some reason using a prototype name instead of the prototype ID (Already a sin) then that seems really really bad.

File: Resources/Prototypes/Entities/Debugging/debug_sweps.yml [hunk 2]
```diff
@@ -34,7 +34,7 @@
           tags:
             - Debug
       gun_chamber:
-        name: Chamber
+        name: chamber
```
Comment (IProduceWidgets) [inline Resources/Prototypes/Entities/Debugging/debug_sweps.yml:37]: These aren't prototype names and might actually break things.

File: Resources/Prototypes/Entities/Debugging/debug_sweps.yml [hunk 3]
```diff
@@ -25,7 +25,7 @@
   - type: ItemSlots
     slots:
       gun_magazine:
-        name: Magazine
+        name: magazine
```
Comment (IProduceWidgets) [inline Resources/Prototypes/Entities/Debugging/debug_sweps.yml:28]: " (I'm certain there are more of these)

**Comment by lzk228** [comment]
gonna open another pr

**Pull request closed without merging**